docs/concepts/00_what_is_hl_aries.md

```diff
@@ -55,3 +57,4 @@ This controller allows for the creation, retrieval, validation and signing of [v
 3. [Hyperledger Aries Wiki](https://wiki.hyperledger.org/display/ARIES/Hyperledger+Aries)
 4. [Aries RFCs - DIDExchange](https://github.com/hyperledger/aries-rfcs/tree/master/features/0023-did-exchange)
 5. [Aries RFCs - Introduce](https://github.com/hyperledger/aries-rfcs/tree/master/features/0028-introduce)
+6. [Aries RFCs - IssueCredential Choreography Diagram](https://github.com/hyperledger/aries-rfcs/tree/master/features/0036-issue-credential#choreography-diagram)
```

Comment:
@soluchok Currently, aries-framework-go supports [Issue Cred V2](https://github.com/hyperledger/aries-rfcs/tree/master/features/0453-issue-credential-v2) protocol, right ?

Reply:
thanks, i've updated it now